myna/application/rve/rve_part_center/postprocess.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2024 Oak Ridge National Laboratory.
+#
+# This file is part of Myna. For details, see the top-level license
+# at https://github.com/ORNL-MDF/Myna/LICENSE.md.
+#
+# License: 3-clause BSD, see https://opensource.org/licenses/BSD-3-Clause.
+#
+from myna.application.rve import RVE
+import os
+import polars as pl
+import yaml
+
+
+def main():
+
+    app = RVE("rve_part_center")
+
+    # Get expected Myna output files
+    step_name = os.environ["MYNA_STEP_NAME"]
+    myna_files = app.settings["data"]["output_paths"][step_name]
+
+    # Set up regions dict if it doesn't exist
+    for part in app.settings["data"]["build"]["parts"]:
+        values = app.settings["data"]["build"]["parts"][part].get("regions")
+        if values is None:
+            app.settings["data"]["build"]["parts"][part]["regions"] = {}
+
+    for myna_file in myna_files:
+        # Get RVE file data
+        df = pl.read_csv(myna_file)
+
+        # Use itertuples to iterate while preserving dtype for all columns
+        # Note: using iterrows creates a pd.Series, which only has one dtype for all values
+        for row in df.iter_rows(named=True):
+            part = str(row["part"])
+            region = f'rve_{row["id"]}'
+            app.settings["data"]["build"]["parts"][part]["regions"][region] = {
+                "X (m)": row["x (m)"],
+                "Y (m)": row["y (m)"],
+                "layer_starts": row["layer_starts"],
+                "layer_ends": row["layer_ends"],
+                "layers": [
+                    x
+                    for x in range(int(row["layer_starts"]), int(row["layer_ends"]) + 1)
+                ],
+            }
+
+    # Update the input file
+    with open(app.input_file, "w") as f:
+        yaml.dump(app.settings, f, sort_keys=False, default_flow_style=None, indent=2)
+
+    # Re-run myna_config to ensure all directories exist if there is a next step
+    os.system(f"myna config --input {app.input_file}")
```

Comment:
Not sure I follow this

Reply:
This re-runs the configuration to add in the data and directory structures for the identified regions, given that the regions are not known a priori. This is necessary if there are any steps afterwards that require the `regions` dictionary in the input file.

Reply:
Blocked by #27 , will update to call config function directly.